### PR TITLE
Update office-365-encryption-for-skype-onedrive-sharepoint-and-exchan…

### PR DESCRIPTION
**Pull request by msbemba**
…ge.md

Added teams to the list with more details

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1651

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/office-365-encryption-for-skype-onedrive-sharepoint-and-exchange.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ ms.collection:
 description: "Summary: A description of encryption for Skype, OneDrive, SharePoint, and Exchange Online."
 ---
 
-# Encryption for Skype for Business, OneDrive for Business, SharePoint Online, and Exchange Online
+# Encryption for Skype for Business, OneDrive for Business, SharePoint Online, Microsoft Teams and Exchange Online
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-encryption-for-skype-onedrive-sharepoint-and-exchange.md]:
```suggestion
# Encryption for Skype for Business, OneDrive for Business, SharePoint Online, Microsoft Teams, and Exchange Online
```

File: microsoft-365/compliance/office-365-encryption-for-skype-onedrive-sharepoint-and-exchange.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ ms.collection:
 description: "Summary: A description of encryption for Skype, OneDrive, SharePoint, and Exchange Online."
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-encryption-for-skype-onedrive-sharepoint-and-exchange.md]:
```suggestion
description: "Summary: A description of encryption for Skype, OneDrive, SharePoint, Microsoft Teams, and Exchange Online."
```

File: microsoft-365/compliance/office-365-encryption-for-skype-onedrive-sharepoint-and-exchange.md [hunk 3]
```diff
@@ -81,3 +81,32 @@ Exchange Online uses BitLocker for all mailbox data, and the BitLocker configura
...
+|Server-to-server|MTLS|
+|Client-to-server (ex. instant messaging and presence)|TLS|
+|Media flows (ex. audio and video sharing of media)|TLS|
+|Audio and video sharing of media|SRTP/TLS|
+|Signaling|TLS|
+|||
+
+#### Media Encryption
+
+Media traffic is encrypted using Secure RTP (SRTP), a profile of Real-Time Transport Protocol (RTP) that provides confidentiality, authentication, and replay attack protection to RTP traffic. SRTP uses a session key generated by using a secure random number generator and exchanged using the signaling TLS channel. Client to Client media traffic is negotiated through a Client to Server connection signaling, but is encrypted using SRTP when going direct Client to Client.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-encryption-for-skype-onedrive-sharepoint-and-exchange.md]:
```suggestion
Media traffic is encrypted using Secure RTP (SRTP), a profile of Real-Time Transport Protocol (RTP) that provides confidentiality, authentication, and replay attack protection to RTP traffic. SRTP uses a session key generated by using a secure random number generator and is exchanged using the signaling TLS channel. Client-to-client media traffic is negotiated through a client-to-server connection signaling but is encrypted using SRTP when going client-to-client directly.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review. This is based on issue #1651.

Thank you.

cc: @kenwith

**Comment by mypil** [comment]
@KCCross - We would like to follow up. Can you please assist to review and merge this PR?

Thank you.

**Pull request merged**